### ISSUE TITLE
Página para Desenvolvedores

### ISSUE THREAD

**Issue by marcelometal** [olhoneles/politicos#13]
Explicar melhor como um desenvolvedores poderá criar seu projeto usando a API

**Comment by gildasio** [comment]
Uma ideia possivelmente interessante é criar a documentação usando a wiki do Github do projeto, o que acha?

Assim separa um pouco as coisas e fica no repositório apenas o código da API.

**Comment by marcelometal** [comment]
Olá @gjuniioor, não gosto dessa ideia de separar, uma vez que as duas coisas andam juntas. Seria melhor distribuir junto. Se a documentação for escrita em Markdown, o github exibe formatado senão me engano.

**Comment by gildasio** [comment]
Sim, feita em Markdown o Github já estiliza no modo dele, mas o django trataria isso para transformar em web? (não saco de django rs').

E fazendo na wiki, isso pode ficar mais aberto para editar, criar páginas e tudo mais, sem que precise a pessoa entender uma tecnologia em específica para criar a documentação.

**Comment by marcelometal** [comment]
@gjuniioor , eu instalei uma app para transformar o markdown em HTML. O website já usa ele, dá uma olhada no [template dos exemplos](https://github.com/olhoneles/politicos/blob/master/website/templates/examples.html).

Penso que num futuro próximo, podemos usar o [MkDocs](http://www.mkdocs.org/) 

Mas se você quiser começar a documentação no wiki do github, não vejo problema, toda contribuição é bem vinda.

**Comment by gildasio** [comment]
Show! Vlw por apresentar esse MkDocs, não conhecia, bacana ele!

Vou tentar puder isso na Wiki e vemos como fica então :)
